src/object/srv_obj_remote.c

```diff
@@ -383,7 +383,7 @@ ds_obj_cpd_clone_reqs(struct daos_shard_tgt *tgt, struct daos_cpd_disp_ent *dcde
 
 				oiod = obj_ec_tgt_oiod_get(split->osr_tgt_oiods,
 							   dcsr_parent[idx].dcsr_ec_tgt_nr,
-							   dcri_parent->dcri_shard_off -
+							   dcri_parent->dcri_shard_id -
```

Comment:
may need confirm with @Nasf-Fan , thx

Reply:
oh, @Nasf-Fan  and I discussed this on PR-9874.